clang/lib/Analysis/UnsafeBufferUsage.cpp

```diff
@@ -781,40 +817,6 @@ struct LibcFunNamePrefixSuffixParser {
   }
 };
 
-// Constant fold a conditional expression 'cond ? A : B' to
```

Comment:
Pulled the two functions out of the `libc_func_matchers ` namespace since they are more general.